Kconfig.zephyr

```diff
@@ -50,8 +50,8 @@ menu "Build and Link Features"
 
 menu "Linker Options"
 
-config KERNEL
-	bool "Include Zephyr kernel"
+config DRIVERS
```

Comment:
Actually, we don't need to have any kconfig like this. Including the `driver` folder works without compilation error now!